docs/submission/submitted-data.md

```diff
@@ -2,103 +2,126 @@
+![Dashbard Donor Data Summary](/assets/submission/dashboard-donor-table-card.png)
 
 #### Table Actions
-Using the buttons above the table, a program member can download all clinical data that has been submitted for the program donors. The download will be in the same file format that was submitted through the sample registration and clinical submission sections. 
 
-*Coming Soon:* A program member can download missing clinical data for all donors as well as the table summary details.
+Using the buttons above the table, a program member can download all clinical data that has been submitted for the program donors. The download will be in the same file format that was submitted through the sample registration and clinical submission sections.
+
+_Coming Soon:_ A program member can download missing clinical data for all donors as well as the table summary details.
 
 The table is sorted by default on the last updated date, so a program member will always see the donors with the most recent activity first. The table can be sorted by another column by clicking on any other column header.
 
 #### Released Files Status
-Each donor has a **star** ![Dashboard Donor Release Star](/assets/submission/dashboard-donor-star.png) that denotes if they have fully released files, partially released files or no released files. In order to have released files, the embargo period must be over and the donor must have 100% [core fields completeness](clinical-data-validation-rules#clinical-data-completeness) as well as some molecular analysis completed.
+
+Each donor has a **star** ![Dashboard Donor Release Star](/assets/submission/dashboard-donor-star.png) that denotes if they have fully released files, partially released files or no released files. In order to have released files, the embargo period must be over and the donor must have 100% [core fields completeness](/docs/submission/clinical-data-validation-rules#clinical-data-completeness) as well as some molecular analysis completed.
 
 #### Clinical Data Submission
-The table shows the percentage of **core clinical fields** and percentage of **extended clinical fields** that have been submitted for a donor. 
 
-![Dashboard Clinical Data Submission](/assets/submission/dashboard-clinical-submission.png) 
-* **The percentage of core fields must be 100% (showing ![Dashboard Donor 100% clinical complete](/assets/submission/dashboard-clinical-complete.png)) for any analysis files to be released to the program members for download**. [See how this percentage is calculated](clinical-data-validation-rules#clinical-data-completeness) 
-* The extended field percentage does not have to reach 100%, but it is strongly encouraged that data submitters provide as many of these fields as possible for a donor.
+The table shows the percentage of **core clinical fields** and percentage of **extended clinical fields** that have been submitted for a donor.
```

Comment:
note...to change if this ticket can be completed: https://github.com/icgc-argo/platform-ui/issues/1456

Reply:
made a ticket for this